src/Npgsql/Internal/Resolvers/AdoTypeInfoResolver.cs

```diff
@@ -228,7 +228,7 @@ static void AddInfos(TypeInfoMappingCollection mappings)
 
         // Time
         mappings.AddStructType<TimeSpan>(DataTypeNames.Time,
-            static (options, mapping, _) => mapping.CreateInfo(options, new TimeSpanTimeConverter()), isDefault: true);
+            static (options, mapping, _) => mapping.CreateInfo(options, new TimeSpanTimeConverter()), MatchRequirement.DataTypeName);
```

Comment:
So which mapping is default for Time now?

Reply:
Ah ok, seems like we never had one for interval either

Reply:
OK. In any case the idea is to continue returning TimeSpan by default for time (since there's nothing else we can), until we do #5328 at which point we'll start returning TimeOnly instead.